src/javatests/com/google/cloud/bigquery/dwhassessment/hooks/MigrationAssessmentLoggingHookTest.java

```diff
@@ -64,99 +66,41 @@ public class MigrationAssessmentLoggingHookTest {
 
   @Mock Hive hiveMock;
 
-  private QueryState queryState;
   private HiveConf conf;
   private String tmpFolder;
-
+  private HookContext hookContext;
+  private QueryState queryState;
 
   @Before
-  public void setup() throws IOException {
-    conf = new HiveConf();
+  public void setup() throws Exception {
+    TestUtils utils = new TestUtils(hiveMock);
+    conf = utils.getConf();
+    queryState = utils.getQueryState();
+    hookContext = utils.getHookContext();
+
     tmpFolder = folder.newFolder().getAbsolutePath();
     conf.set(LoggerVarsConfig.HIVE_QUERY_EVENTS_BASE_PATH.getConfName(), tmpFolder);
-    queryState = new QueryState(conf);
+  }
+
+  @After
+  public void tearDown() {
```

Comment:
IIUC `@Before` is executed on every test. If that's so, `@After` which cleans fields in mutable fields is not needed